### PR TITLE
Update LLM-Connections.md

### PR DESCRIPTION
**Pull request by selimhanerhan**
Created a short documentation on how to integrate crewAI and Llama2 locally with the help of Ollama. These steps might be self explanatory for the advanced programmers but will be definitely needed for the beginners like myself, so I decided to add it into the documentations to have a free alternative to the OpenAI.

### CODE REVIEW

File: docs/how-to/LLM-Connections.md [hunk 1]
```diff
@@ -50,6 +50,71 @@ OPENAI_MODEL_NAME='openhermes'  # Adjust based on available model
...
+            tasks=[task],
+            verbose=2
+        )
+
+result = crew.kickoff()
+
+print(result)
+```
+
+
```
Comment (shivamjagdishsingh) [inline docs/how-to/LLM-Connections.md]: Check out my stackoverflow answer to run pre-commit on last commit.
https://stackoverflow.com/a/78196222/12821733

Comment (selimhanerhan) [reply docs/how-to/LLM-Connections.md]: Thank you! I was wondering why it was failing lint and all other checks.

**Review by selimhanerhan — COMMENTED** [review]
Now detailed Ollama setup is in the documentation so Note is not needed

**Review by joaomdmoura — APPROVED** [review]
Thanks! Great imrpovements

**Pull request merged**